tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -1053,12 +1053,14 @@ def test_positive_find_admin_user(self):
 
     @pytest.mark.skip_if_not_set('libvirt')
     @pytest.mark.tier4
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_match('8')
```

Comment:
Why only 8?

Reply:
ping @pondrejk

Reply:
@lhellebr @JacobCallahan well, eventually I reverted back to 7, there is a lot to improve in these tests and I don't give up, but I'd like to merge this pr to get them passing again. I'm working on a follow-up pr to (a) allow running against multiple hosts (currently precluded by the way repos are enabled in these tests), (b) make it use new style of host registration (register function doesn't currently play well with org dict returned from cli helper), etc. etc.